bemani/utils/proxy.py

```diff
@@ -154,7 +154,7 @@ def receive_request(path: str) -> Response:
         )
 
     # Set up custom headers for remote request
-    headers = {}
+    headers = {key: value for key, value in request.headers.items()}
```

Comment:
All of the headers that are needed for a successful XRPC request are set below in the if statements.